ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -0,0 +1,531 @@
+      <Button style={{marginLeft: '0.5rem'}} onClick={() => selfBypass()}>Bypass all</Button>
+    </FlexRow>;
+  };
+
+  const ModulesForCard = (props: {modules: AccessModule[]}) => {
+    const {modules} = props;
+    return <FlexColumn style={styles.modulesContainer}>
+      {modules.map(module =>
+          <MaybeModule key={module} module={module} active={module === activeModule}/>
+      )}
+    </FlexColumn>;
+  };
+
+  const RegisteredTierCard = () => {
+    return <FlexRow style={styles.card}>
+      <FlexColumn>
+        <div style={styles.cardStep}>Step 1</div>
+        <div style={styles.cardHeader}>Complete Registration</div>
+        <FlexRow>
+          <RegisteredTierBadge/>
+          <div style={styles.rtData}>Registered Tier data</div>
+        </FlexRow>
+        <div style={styles.rtDataDetails}>Once registered, you’ll have access to:</div>
+        <FlexRow>
+          <DARIcons.individual style={styles.rtDetailsIcon}/>
+          <div style={styles.rtDataDetails}>Individual (not aggregated) data</div>
+        </FlexRow>
+        <FlexRow>
+          <DARIcons.identifying style={styles.rtDetailsIcon}/>
+          <div style={styles.rtDataDetails}>Identifying information removed</div>
+        </FlexRow>
+        <FlexRow>
+          <DARIcons.electronic style={styles.rtDetailsIcon}/>
+          <div style={styles.rtDataDetails}>Electronic health records</div>
+        </FlexRow>
+        <FlexRow>
+          <DARIcons.survey style={styles.rtDetailsIcon}/>
+          <div style={styles.rtDataDetails}>Survey responses</div>
+        </FlexRow>
+        <FlexRow>
+          <DARIcons.physical style={styles.rtDetailsIcon}/>
+          <div style={styles.rtDataDetails}>Physical measurements</div>
+        </FlexRow>
+        <FlexRow>
+          <DARIcons.wearable style={styles.rtDetailsIcon}/>
+          <div style={styles.rtDataDetails}>Wearable devices</div>
+        </FlexRow>
+      </FlexColumn>
+      <ModulesForCard modules={rtModules}/>
+    </FlexRow>;
+  };
+
+  const DuccCard = () => <FlexRow style={{...styles.card, height: '125px'}}>
+    <FlexColumn>
+      {/* This will be Step 3 when CT becomes the new Step 2 */}
+      <div style={styles.cardStep}>Step 2</div>
+      <div style={styles.cardHeader}>Sign the code of conduct</div>
+    </FlexColumn>
+    <ModulesForCard modules={duccModule}/>
+  </FlexRow>;
+
+  const {config: {unsafeAllowSelfBypass}} = useStore(serverConfigStore);
+
+  return <FlexColumn style={styles.pageWrapper}>
+    <DARHeader/>
+    {profile && !activeModule && <Completed/>}
+    {unsafeAllowSelfBypass && activeModule && <SelfBypass/>}
+    <FadeBox style={styles.fadeBox}>
+      <div style={styles.pleaseComplete}>
+        Please complete the necessary steps to gain access to the <AoU/> datasets.
+      </div>
+      <RegisteredTierCard/>
+      {/* TODO RW-7059 - Step 2 ControlledTierCard */}
+      <DuccCard/>
+    </FadeBox>
+    </FlexColumn>;
```

Comment:
nit: I think I would indent more, or indent less somewhere here, doesn't feel like thses two should be aligned